master-firmware/tests/test_strategy_helpers.cpp

```diff
@@ -12,25 +12,94 @@ TEST_GROUP(ABlockPositionComputer)
     }
 };
 
-TEST(ABlockPositionComputer, FindsPositionOfTrivialBlocksPose)
+TEST(ACubePositionComputer, FindsPositionOfTrivialBlocksPose)
 {
-    se2_t blocks_pose = se2_create_xya(0, 0, RADIANS(0));
+    se2_t cubes_pose = se2_create_xya(0, 0, RADIANS(0));
 
-    POINT_EQUAL({  0,   0}, strategy_block_pos(blocks_pose, BLOCK_YELLOW));
-    POINT_EQUAL({ 60,   0}, strategy_block_pos(blocks_pose, BLOCK_GREEN));
-    POINT_EQUAL({  0,  60}, strategy_block_pos(blocks_pose, BLOCK_BLUE));
-    POINT_EQUAL({-60,   0}, strategy_block_pos(blocks_pose, BLOCK_RED));
-    POINT_EQUAL({  0, -60}, strategy_block_pos(blocks_pose, BLOCK_BLACK));
+    POINT_EQUAL({  0,   0}, strategy_cube_pos(cubes_pose, CUBE_YELLOW));
+    POINT_EQUAL({ 60,   0}, strategy_cube_pos(cubes_pose, CUBE_GREEN));
+    POINT_EQUAL({  0,  60}, strategy_cube_pos(cubes_pose, CUBE_BLUE));
+    POINT_EQUAL({-60,   0}, strategy_cube_pos(cubes_pose, CUBE_RED));
+    POINT_EQUAL({  0, -60}, strategy_cube_pos(cubes_pose, CUBE_BLACK));
 }
 
-TEST(ABlockPositionComputer, FindsPositionOfNonTrivialBlocksPose)
+TEST(ACubePositionComputer, FindsPositionOfNonTrivialBlocksPose)
 {
-    se2_t blocks_pose = se2_create_xya(100, 200, RADIANS(45));
+    se2_t cubes_pose = se2_create_xya(100, 200, RADIANS(45));
     float offset = 60.f / sqrtf(2);
 
-    POINT_EQUAL({100,          200         }, strategy_block_pos(blocks_pose, BLOCK_YELLOW));
-    POINT_EQUAL({100 + offset, 200 + offset}, strategy_block_pos(blocks_pose, BLOCK_GREEN));
-    POINT_EQUAL({100 - offset, 200 + offset}, strategy_block_pos(blocks_pose, BLOCK_BLUE));
-    POINT_EQUAL({100 - offset, 200 - offset}, strategy_block_pos(blocks_pose, BLOCK_RED));
-    POINT_EQUAL({100 + offset, 200 - offset}, strategy_block_pos(blocks_pose, BLOCK_BLACK));
+    POINT_EQUAL({100,          200         }, strategy_cube_pos(cubes_pose, CUBE_YELLOW));
+    POINT_EQUAL({100 + offset, 200 + offset}, strategy_cube_pos(cubes_pose, CUBE_GREEN));
+    POINT_EQUAL({100 - offset, 200 + offset}, strategy_cube_pos(cubes_pose, CUBE_BLUE));
+    POINT_EQUAL({100 - offset, 200 - offset}, strategy_cube_pos(cubes_pose, CUBE_RED));
+    POINT_EQUAL({100 + offset, 200 - offset}, strategy_cube_pos(cubes_pose, CUBE_BLACK));
+}
+
+TEST_GROUP(ABlockPositionComputer)
+{
+    se2_t poses[4] = {
+        se2_create_xya(850 - 160, 540 - 160, -45),
+        se2_create_xya(850 + 160, 540 - 160,  45),
+        se2_create_xya(850 + 160, 540 + 160, 135),
+        se2_create_xya(850 - 160, 540 + 160, 225),
+    };
+
+    void SE2_EQUAL(se2_t lhs, se2_t rhs, double tolerance=0.01)
```

Comment:
Idea: We could create a set of cpputest utilities for use with our custom datatypes. Summer project probably.